### DISCUSSION TITLE
No longer can create new layouts unless signed in

### DISCUSSION THREAD

**Discussion by bertaveira** [foxglove/community#911] [Q&A] [answered]
I am wondering why in recent versions you no longer can create a new layout? The button to create a new layout is greyed out unless the user signs in.

Previously created layouts still exist and users can still import layouts from files but not create new ones. Since in some applications (for example shared computers) adding an account is impractical and not beneficial I wonder why the need to be a registered user to take advantage of a core feature of foxglove studio like layouts.

**Comment by amacneil** [comment, marked as answer]
For multi-layouts specifically, this is a feature of our closed source release. We have continued (and will continue) to make that feature freely available, but explicitly linking layouts to an account allows us to improve integration with our backend and simplify the sync logic.

More generally, we're moving towards adding full support for browsing cloud data in our desktop app, and will eventually require sign in too. For a shared computer it is not uncommon to create a dedicated service account to sign in from there.

**Reply by bertaveira** [reply]
Quite understandable to have to fund the development team. 

Still leaves me with sadness to follow the progress of Foxglove since webviz days and how it changed the open source visualisation and data handling space and now getting more closed source by the day. I fear in a few years time we will be in the same place we were before foxglove, with no extendable and modifiable data visualization platform open sourced to the public. I just noticed today custom extensions are only for paying customers which unfortunately means I probably won't be using it as much for my future projects.

I've been applying foxglove to out of the ordinary applications which means they normally require some modifications and custom made solutions which would never be aded or supported with regular Foxglove. I guess this path will make foxglove amazing for customers in the sector it targets but all others will not be able to use it and modify it as they have in the past.

**Reply by defunctzombie** [reply]
> I just noticed today custom extensions are only for paying customers

Where are you seeing this? The intent is that users of the free app can also use custom extensions. You do not need a paid plan to use the extension feature. If you are seeing otherwise we can correct that.
